bigtable/integration_test.go

```diff
@@ -2480,77 +2507,107 @@ func TestIntegration_AdminBackup(t *testing.T) {
 
 	// Create backup
 	backupName := "mybackup"
-	defer adminClient.DeleteBackup(ctx, cluster, "mybackup")
 
-	if err = adminClient.CreateBackup(ctx, table, cluster, backupName, time.Now().Add(8*time.Hour)); err != nil {
-		t.Fatalf("Creating backup: %v", err)
+	if err = adminClient.CreateBackup(ctx, tblConf.TableID, sourceCluster, backupName, time.Now().Add(8*time.Hour)); err != nil {
+		t.Errorf("Creating backup: %v", err)
```

Comment:
I still think that this and some of the following should be changed to `Fatalf`. For example, if the backup is not successfully created, will the checks below be useful?